twiggy/tests/all/expectations/dominators_json_prints_multiple_root_items

```diff
@@ -0,0 +1 @@
+{"items":[{"name":"calledOnce","shallow_size":5,"shallow_size_percent":3.4722222222222223,"retained_size":5,"retained_size_percent":3.4722222222222223},{"name":"calledTwice","shallow_size":5,"shallow_size_percent":3.4722222222222223,"retained_size":5,"retained_size_percent":3.4722222222222223}]}
```

Comment:
noticed a problem when arguments were given to this subcommand, where the fields would all be shown side by side. This was interfering with the summary output, so I fixed that in this PR :)